assets/panel/debugger.properties

```diff
@@ -466,7 +466,7 @@ welcome.search=%S to search for sources
 
 # LOCALIZATION NOTE (sourceSearch.search): The center pane Source Search
 # prompt for searching for files.
-sourceSearch.search=Search Sources…
```

Comment:
hmm - what do others think I like this new name.

If we change it we should bump the key to `sourceSearch.search2`